README.md

```diff
@@ -1,5 +1,21 @@
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
+### Build for Deploy
+
+```shell
+$ npm run build
+# or
+$ yarn build
+```
```

Comment:
We might need a section of export static files for deployment as we are currently using S3 to store this web app.

Reply:
Have a look at cc-app repo

Reply:
got it, I will add this one